scipy/sparse/linalg/isolve/lsmr.py

```diff
@@ -88,7 +88,7 @@ def lsmr(A, b, damp=0.0, atol=1e-6, btol=1e-6, conlim=1e8,
     show : bool, optional
         Print iterations logs if ``show=True``.
     x0 : array_like, shape (n,), optional
-        Initial guess of x, if None zeros are used.
+        Initial guess of ``x``, if None zeros are used.
 
         .. versionadded:: 1.0.0
     Returns
```

Comment:
Do you mind also adding a line break before `Returns` here so that it is registered as a section?